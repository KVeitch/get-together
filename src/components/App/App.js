
import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { Route } from 'react-router-dom';
import { bindActionCreators } from 'redux';
import { connect } from 'react-redux';

import './App.scss';

import logo from '../../logo.svg'

export class App extends Component {
  async componentDidMount() {

  }


  render = () => {
    return (
      <div className="App">
        <Route to='/'/>
        <Route to='/'/>

      </div>
      
    );
  }
}

export const mapStateToProps = (state) => ({

});

export const mapDispatchToProps = (dispatch) => (bindActionCreators({
  
}, dispatch));

export default connect(mapStateToProps, mapDispatchToProps) (App);
