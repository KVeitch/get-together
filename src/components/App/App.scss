@import '../../variables.scss';
// $main-bg: #d6cbff;
// $main-higlight: #CBDEFF;
// $main-secondary: #FFCBDA

* {
  font-family: 'Source Sans Pro', sans-serif;
  margin:0;
  padding:0;
  box-sizing: border-box;

}

.App {
  height:100vh;
  width:100vw;
  text-align: center;
  background-color: white;
  
}

.title {
  height:14vh;
  padding:3vh 0;
  background-color: $main-higlight;
  border-bottom: 1px solid $main-secondary;
  display:flex;
  justify-content: center;

  .title-container{
    display:flex;
    justify-content: center;
    width:40vw;
  }

  a {
    text-decoration: none;
  }

  h1 {
    font-size: 2.4em;
    color:$logo-color;
    margin:0 .5vw;
  }

  .plane-image {
    height:15vh;
  }

  .image-mirror{
    -webkit-transform: scaleX(-1);
    transform: scaleX(-1);
  }
}

