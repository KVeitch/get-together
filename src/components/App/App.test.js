import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  const wrapper = shallow(<App />);

  it('should match the snapshot with all the data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
