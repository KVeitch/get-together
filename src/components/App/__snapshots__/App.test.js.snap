// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot with all the data passed in correctly 1`] = `
<div
  className="App"
>
  <header
    className="title"
  >
    <div
      className="title-container"
    >
      <Link
        to="/"
      >
        <img
          alt="Line art of a blue plane flying into a cloud"
          className="plane-image"
          src="blue-plane.png"
        />
      </Link>
      <Link
        to="/"
      >
        <h1>
          Connection
        </h1>
      </Link>
      <Link
        to="/"
      >
        <img
          alt="Line art of a blue plane flying into a cloud"
          className="plane-image image-mirror"
          src="blue-plane.png"
        />
      </Link>
    </div>
  </header>
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/trip"
    render={[Function]}
  />
</div>
`;
